packages/block-library/src/comments/index.php

```diff
@@ -76,6 +76,7 @@ function render_block_core_comments( $attributes, $content, $block ) {
 	 * why they are not defined in `block.json`.
 	 */
 	wp_enqueue_script( 'comment-reply' );
+	wp_script_add_data( 'comment-reply', 'strategy', 'defer' ); // TODO: For core merge, this would rather be done in wp-includes/script-loader.php.
```

Comment:
I understand this is experimental, but I find it also rather pointless, given that the `comment-reply` script already appears in the footer, so would barely get any benefit from deferring it. I think we should discuss this change elsewhere, as it also doesn't really fit into the scope of this PR, given everything else is focused on block scripts.
```suggestion
```